code/game/objects/items/stacks/stack.dm

```diff
@@ -355,7 +357,11 @@
 	use(amount)
 
 /obj/item/stack/attackby(obj/item/W, mob/user, params)
-	if(istype(W, merge_type))
+	if(!parent_stack && istype(W, merge_type))
+		var/obj/item/stack/S = W
+		merge(S)
+		to_chat(user, "<span class='notice'>Your [S.name] stack now contains [S.get_amount()] [S.singular_name]\s.</span>")
+	if(parent_stack==TRUE && W.type == type)
```

Comment:
```suggestion
	if(parent_stack && W.type == type)
```

Reply:
Fixed that! Thanks for checking it so quickly!